tests/integration/test_charm_one_runner.py

```diff
@@ -49,6 +50,35 @@ async def test_network_access(app: Application) -> None:
     assert stdout is None
 
 
+@pytest.mark.asyncio
+@pytest.mark.abort_on_fail
+async def test_usage_of_aproxy(model: Model, app_no_runner: Application, squid_proxy: str) -> None:
+    """
+    arrange: A working application with one runner using aproxy configured for a proxy server.
+    act: Run curl in the runner.
+    assert: The aproxy log contains the request.
+    """
+    app = app_no_runner  # Rename to make it clear that the app will contain a runner.
```

Comment:
Perhaps create a different fixture? How would this runner be added in this case?